dev-qt/qttools/qttools-6.5.1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qt6-build
+
+DESCRIPTION="Qt Tools Collection"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+IUSE="
+	assistant designer distancefieldgenerator +linguist pixeltool
+	qdbus qdoc qtattributionsscanner qtdiag qtplugininfo
+"
+
+DEPEND="
+	=dev-qt/qtbase-${PV}*[network]
+	assistant? ( =dev-qt/qtbase-${PV}*[sql,widgets] )
+	designer? ( =dev-qt/qtbase-${PV}*[widgets] )
+	distancefieldgenerator? (
+		=dev-qt/qtbase-${PV}*[widgets]
+		=dev-qt/qtdeclarative-${PV}*
+	)
+	pixeltool? ( =dev-qt/qtbase-${PV}*[widgets] )
+	qdbus? ( =dev-qt/qtbase-${PV}*[widgets] )
+	qdoc? ( sys-devel/clang:= )
+	qtdiag? ( =dev-qt/qtbase-${PV}*[opengl,widgets] )
```

Comment:
this would seem to be at odds with Qt proj's no `qt` prefix policy.

Reply:
Changed for a couple of other ones too.